test/unit/dropbox.js

```diff
@@ -86,6 +99,26 @@ describe('Dropbox', () => {
       chai.assert.equal(APP_AUTH, dbx.rpcRequest.getCall(0).args[2]);
     });
 
+    it('completes a cookie auth RPC request when not supplied an access token', () => {
+      const dbxAuth = new DropboxAuth();
+      const dbx = new Dropbox({ auth: dbxAuth });
+      const rpcSpy = sinon.spy(dbx, 'rpcRequest');
+      dbx.request('path', {}, COOKIE, 'api', RPC)
+        .catch((error) => {
+          fail(error);
+        });
+      chai.assert.isTrue(rpcSpy.calledOnce);
+      chai.assert.equal('path', dbx.rpcRequest.getCall(0).args[0]);
+      chai.assert.deepEqual({}, dbx.rpcRequest.getCall(0).args[1]);
+      chai.assert.equal(COOKIE, dbx.rpcRequest.getCall(0).args[2]);
+    });
+
+    it('errors if cookie auth is supplied an OAuth token', () => {
```

Comment:
Not sure if this is actually the behavior we want for cookie auth but can remove if not.